daemon/cmd/daemon_main.go

```diff
@@ -648,12 +648,19 @@ func initializeFlags() {
 	option.BindEnv(option.EnableHostFirewall)
 
 	flags.String(option.NativeRoutingCIDR, "",
-		fmt.Sprintf("Allows to explicitly specify the IPv4 CIDR for native routing. This value corresponds to the configured cluster-cidr. Deprecated in favor of --%s", option.IPv4NativeRoutingCIDR))
+		fmt.Sprintf("Allows to explicitly specify the IPv4 CIDR for native routing."+
+			"When specified, Cilium assumes something else has configured the networking for this range and will hand traffic destined to that range to the Linux network stack without applying any SNAT. "+
```

Comment:
```suggestion
			"When specified, Cilium assumes networking for this CIDR is preconfigured and it will hand traffic destined to that range to the Linux network stack without applying any SNAT. "+
```

Reply:
One addendum to @kkourt's suggestion above, wherever possible try to avoid "it" as this can often add ambiguity to statements. `it` -> `Cilium` will clarify further.

Reply:
@joestringer isn't two consecutive Cilium's too much?

Maybe:
"When specified, Cilium assumes networking for this CIDR is preconfigured and hands traffic destined for that range to the Linux network stack without applying any SNAT. "